censustree/censustree.go

```diff
@@ -1,27 +1,88 @@
 package censustree
 
 import (
+	"fmt"
+	"path/filepath"
+	"strings"
 	"sync/atomic"
 
+	"github.com/vocdoni/arbo"
 	"go.vocdoni.io/dvote/db"
+	"go.vocdoni.io/dvote/db/badgerdb"
 	"go.vocdoni.io/dvote/tree"
+	"go.vocdoni.io/proto/build/go/models"
+)
+
+const (
+	// TreeUnknown is the default value used for censusTree implementations
+	// which are not part of the this factory.
+	TreeTypeUnknown = models.Census_UNKNOWN
```

Comment:
the qualified name here will be `censustree.TreeTypeUnknown`, so I think you can drop the redundant "Tree" and just make it `censustree.TypeUnknown`. Same with the others.

(this problem exists elsewhere, e.g. censustree.CensusTree is very redundant, but that's in master)

Reply:
Right, updated both things that you mention:
- `TreeTypeUnknown` -> `TypeUnknown` (the same with the other types)
- `CensusTree` -> `Tree`